packages/blocks/package.json

```diff
@@ -9,7 +9,7 @@
     "directory": "packages/blocks"
   },
   "scripts": {
-    "tsc": "tsc -p ./tsconfig.build.json",
+    "tsc": "tsc -b ./tsconfig.build.json",
```

Comment:
-b is for incremental builds https://www.typescriptlang.org/docs/handbook/project-references.html